### PR TITLE
Update New-DlpComplianceRule.md

### PR DESCRIPTION
**Pull request by ybhargav1995**

### CODE REVIEW

**Comment by chrisda** [comment]
@ybhargav1995, why remove the description?

**Comment by ybhargav1995** [comment]
> @ybhargav1995, why remove the description?
Because the params FromMemberOf and ExceptIfFromMemberOf are no longer for internal use. It is available for the customers

File: exchange/exchange-ps/exchange/New-DlpComplianceRule.md [hunk 1]
```diff
@@ -1180,7 +1180,7 @@ Accept wildcard characters: False
 ```
 
 ### -ExceptIfFromMemberOf
-This parameter is reserved for internal Microsoft use.
+The ExceptIfFromMemberOf parameter specifies an exception for the DLP rule that looks for senders' email in the lists given as values to this param. You can give multiple lists to this parameter.
```
Comment (JohanFreelancer9) [inline exchange/exchange-ps/exchange/New-DlpComplianceRule.md:1183]:
```suggestion
The ExceptIfFromMemberOf parameter specifies an exception for the DLP rule that looks for senders' emails in the lists given as values to this parameter. You can give multiple lists to this parameter.
```

File: exchange/exchange-ps/exchange/New-DlpComplianceRule.md [hunk 2]
```diff
@@ -1842,7 +1842,7 @@ Accept wildcard characters: False
 ```
 
 ### -FromMemberOf
-This parameter is reserved for internal Microsoft use.
+The FromMemberOf parameter specifies an condition for the DLP rule that looks for senders' email in the lists given as values to this param. You can give multiple lists to this parameter.
```
Comment (JohanFreelancer9) [inline exchange/exchange-ps/exchange/New-DlpComplianceRule.md:1845]:
```suggestion
The FromMemberOf parameter specifies a condition for the DLP rule that looks for senders' emails in the lists given as values to this parameter. You can give multiple lists to this parameter.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @ybhargav1995, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**